s3tests_boto3/functional/test_iam.py

```diff
@@ -0,0 +1,440 @@
+import json
+from operator import eq
+
+from botocore.exceptions import ClientError
+from nose.plugins.attrib import attr
+
+from s3tests.functional.utils import assert_raises
+from . import get_tenant_iam_client, get_tenant_user_id
```

Comment:
Let's try and follow the import conventions followed in s3tests_boto3/functional/test_s3.py